package.json

```diff
@@ -185,7 +185,7 @@
     "valid-url": "^1.0.9",
     "vreme": "^3.0.2",
     "web3": "^0.20.1",
```

Comment:
Ok, @matthewlilley , (please take into account that I've still no full understanding of the overall tasks towards w3 1.0 support) I cannot review your code, but shouldn't this `"web3": "^0.20.1"` become `"web3": "1.0.0"` something? Or does your work only provide a subset of the full web3 1.0 support?

Reply:
@lazaridiscom The latter. 

I assume it can also be updated internally, but I consciously left it out for now. I'll update the description and amend a couple of small changes soon. If you can get an answer on updating it internally, I'll be happy to do that as well.

Reply:
@lazaridiscom You can use this repo to test.

https://github.com/matthewlilley/metamask-web3-1.0-subscriptions-test

Reply:
oh! you must always assume "reduced mental capacity" of testers. This means that you should provide instructions (ideally a command line sequence)

Reply:
(see, e.g. me currently at around 10%, barely capable of following some instructions. On the first barrier, I would stop)

Reply:
git clone https://github.com/matthewlilley/metamask-web3-1.0-subscriptions-test
cd metamask-web3-1.0-subscriptions-test
npm start

Reply:
You'll need to build the extension and add it to Chrome as well. I gave you instructions on how to do that in a previous thread.

Reply:
(super-simple, thanks! )

[pj.code.docu] Always provide basic instructions, even for the most trivial things. Example: you provide a test-repo, with trivial code. Add a README with the instructions, ideally command-line.

Reply:
I think the web3 version should actually remain `0.20.1`. The fact that MetaMask injects Web3 into the page is a legacy feature.

Well-written dapps won't use the Web3 instance that MetaMask injects, and will instead include their chosen version of Web3 (or indeed an alternative library) in their Javascript.

Poorly written dapps will assume that MetaMask injects a Web3 instance that they can just use, and that will always work in the same way, and will be confused when Web3 1.0 makes backwards-incompatible changes to APIs that worked last week.

Either way, I don't think anyone will want Web3 1.0 injected into the browser.

Reply:
@jamespic That was my understanding too, which is part of the reason I left it out.

Reply:
i guess I'll never get it. (@danfinlay & @kumavis , not sure if adding you to reviewers pings you)

[pj.code.review] - Be aware that delayed reviews can lead to frustration and demoralization of contributors. Try to review early and to announce delays. In context of bounties: its an (unwritten) contract/agreement, and one of your obligations is to provide immediate(!) feedback/reviews.